app/controllers/WorkerController.php

```diff
@@ -81,4 +81,27 @@ public function registerAction(): void
 
         $this->view->errors = $errors;
     }
+
+    /**
+     * If admin is logged in, updates worker via worker service.
+     */
+    public function editAction(): void
+    {
+        if (false === $this->authenticationService->isLoggedIn()) {
```

Comment:
authentication seems to be on every controller and repeating. this seems too look like a middleware